test/Reflection/typeref_decoding.swift

```diff
@@ -10,7 +10,6 @@
 
 // FIXME: rdar://127796117
 // UNSUPPORTED: OS=linux-gnu && CPU=aarch64
```

Comment:
Do we still need this marked as unsupported on Linux?

Reply:
Same as above.

Reply:
[Same passing claim as above](https://github.com/swiftlang/swift/pull/82325#issuecomment-3002136570), can probably be checked by enabling them and running the linux AArch64 CI on this pull.